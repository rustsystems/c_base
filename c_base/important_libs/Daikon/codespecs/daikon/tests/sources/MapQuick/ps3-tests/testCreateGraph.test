#Test creating a graph
CreateGraph A
