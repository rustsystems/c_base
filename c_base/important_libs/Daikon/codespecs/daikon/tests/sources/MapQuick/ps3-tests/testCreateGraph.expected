#Test creating a graph
created graph A
